3rdParty/velocypack/include/velocypack/Builder.h

```diff
@@ -210,7 +210,7 @@ class Builder {
       throw Exception(Exception::InternalError, "Cannot move an open Builder");
     }
     if (this != &that) {
-      _buffer = that._buffer;
+      _buffer = std::move(that._buffer);
```

Comment:
This I don't like. This will mean that the moved-from `Builder` is unsafe to use from this point on.
_unsafe_ in the sense of that it may crash when used.